decrypt.go

```diff
@@ -0,0 +1,20 @@
+package cogs
+
+import (
+    "fmt"
+    "go.mozilla.org/sops/v3/decrypt"
+    "gopkg.in/yaml.v3"
+)
+
+func decryptFile(path string) (map[string]string, error) {
+    yamlMap := make(map[string]string)
+    sec, err := decrypt.File(path, "yaml")
+    if err != nil {
+        return nil, fmt.Errorf("cannot decrypt file: %s", err)
+    }
+    err = yaml.Unmarshal(sec, &yamlMap)
```

Comment:
We should unmarshal  it to a `byte[]` that way we can call `NewYamlVisitor` to and use `yamlVisitor.SetValue`:
https://github.com/Bestowinc/cogs/blob/add_env_path/file.go#L67-L81

This should allow us to to handle dotenv files as well so long as we keep decrypting it to yaml with `decrypt.File(path, "yaml")`